File: materijali/1. godina/p2/cas1/7.c
/* Napisati program koji ispisuje broj navedenih argumenata
komadne linije, a zatim i same argumente sa rednim brojevima. */

#include <stdio.h>

int main(int argc, char* argv[]){
 printf("%d\n", argc);
 
 int i;
 for(i=0; i<argc; i++){
  printf("%d. %s\n", i, argv[i]);
 }
 
 return 0;
}

File: 3. godina/pp vezbe/cas01 - Skript programiranje (Python)/7.py
# Napisati program koji ilustruje osnovne sistemske funkcije i module
# programskog jezika Python tako sto koristi funkcije faktorijel, logaritam, 
# generise pseudoslucajne brojeve, ispisuje argumente komandne linije na izlaz,
# poziva sistemski poziv listanja tekuceg direktorijuma, ispisuje broj sekundi
# od 1.1.1970. godine, finalno unosi string sa ulaza za koji proverava da li 
# predstavlja zapis celog broja.

import math
print(math.factorial(5))
print(math.log(5, 4))

import random
print(random.random())
print(random.randint(0, 10))

import sys
print(sys.argv)

import os
os.system("ls -l")

import time as t
print(t.time())

from re import search as re_src
print(re_src(r'[0-9]+', input("Unesite string za match: ")))